src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -51,15 +51,27 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$, breadcrumbsAppendEx
     ),
   }));
 
-  if (breadcrumbsAppendExtension) {
-    const lastCrumb = crumbs[crumbs.length - 1];
-    lastCrumb.text = (
-      <>
-        {lastCrumb.text}
-        <HeaderExtension extension={breadcrumbsAppendExtension.content} display={'inlineBlock'} />
-      </>
-    );
-  }
+  const Breadcrumbs = (
+    <EuiHeaderBreadcrumbs
+      breadcrumbs={crumbs}
+      max={10}
+      data-test-subj="breadcrumbs"
+      className={classNames({
+        'chrHeaderNavBreadcrumbs--withAppendExtension': !!breadcrumbsAppendExtension,
```

Comment:
If there is a conditional modifier class of `chrHeaderNavBreadcrumbs--withAppendExtension` for this component, I imagine we'll also want an always-present `chrHeaderNavBreadcrumbs` base class as well.

Reply:
Thanks!
Not relevant with new approach